nth-prime.json

```diff
@@ -0,0 +1,34 @@
+{
+  "#": [
+    "The tests that expect 'false' should be implemented to raise",
+    "an error, or indicate a failure. Implement this in a way that",
+    "makes sense for your language."
+  ],
+  "cases": [
+    {
+       "description": "first",
+       "input": "1",
+       "expected": "2"
+    },
+    {
+       "description": "second",
+       "input": "2",
+       "expected": "3"
+    },
+    {
+       "description": "sixth prime",
+       "input": "6",
+       "expected": "13"
+    },
+    {
+       "description": "big prime",
+       "input": "10_001",
+       "expected": "104_743"
+    },
+    {
+       "description": "weird case",
```

Comment:
I would simply remove the "weird case" and leave each implementation to decide how to deal with it. That would probably be better for languages that can enforce some restrictions in the type system.